packages/catalog-api/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@webcomponents/catalog-api",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Schema and interfaces for the @webcomponents/catalog-server",
+  "author": "Google LLC",
+  "homepage": "",
+  "license": "APACHE-2.0",
```

Comment:
Are we doing Apache or BSD?

Reply:
I'm not entirely sure... let's chat?

Reply:
The old site is using Apache: https://github.com/webcomponents/webcomponents.org/blob/old-site/LICENSE and I copied the license over to the main branch in the initial commit.

But this is all new code, so I'm not sure if we need to stick with that. It seems easier though.

Reply:
Since Lit will be in our bundles, it would be very slightly preferable for compression to have the same license in the client code that we ship. And I think it simplifies this entire repo if the entire thing has one license, because otherwise we need multiple LICENSE files, etc.